mongodbatlas/data_source_mongodbatlas_shared_tier_snapshot.go

```diff
@@ -0,0 +1,93 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// This datasource does not have a resource: we tested it manually
+func dataSourceMongoDBAtlasSharedTierSnapshot() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceMongoDBAtlasSharedTierSnapshotRead,
+		Schema: map[string]*schema.Schema{
```

Comment:
FYI - We could easily generate initial schemas for developers to adapt them in the future. 
We have that capability in the SDK - just need to adjust it to your needs.

Reply:
Could you share more details about it? A link to the SDK codebase where I can look at would be enough. Thanks!

Reply:
I missed that comment.
We have a doc for using custom templates:

https://github.com/mongodb/atlas-sdk-go/blob/main/tools/CONTRIBUTING.md#generating-custom-files

We can generate some level of boilerplate for the team like TF schemas.
We can release it under separate module or package.